dynatrace_base/tests/test_dynatrace_base.py

```diff
@@ -0,0 +1,36 @@
+# (C) StackState 2021
```

Comment:
change filename to `test_dynatrace_client` to maintain consistency.

Reply:
Done